target/linux/pistachio/base-files/etc/uci-defaults/02_enable_bt

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+ln -s /etc/init.d/bt-csr /etc/rc.d/S22bt-csr
+sleep 2
+/etc/rc.d/S22bt-csr start
```

Comment:
Won't the scripts from init.d folder will get executed on boot up ?
why do we need to explicitly call it ?